src/test/java/org/traccar/protocol/Xrb28ProtocolDecoderTest.java

```diff
@@ -27,6 +27,12 @@ public void testDecode() throws Exception {
 
         verifyPosition(decoder, text(
                 "*SCOR,OM,863158022988725,D0,0,124458.00,A,2237.7514,N,11408.6214,E,6,0.21,151216,10,M,A#"));
+           
+         verifyPosition(decoder, text(
```

Comment:
Wrong indentation. Also, I think two tests are completely unnecessary here. Once should be enough. Also, use a sample from real device, not just manually modified sample from other test.